sponsors.html

```diff
@@ -1,97 +1,145 @@
+      </div>
+      <div class="section">
+        <h2 class="section-title">Platinum Tier</h2>
+        <div class="sponsors">
+          <div class="sponsor">
+            <h3 class="sponsor-name">CJT Financial Inc.</h3>
+            <img
+              class="sponsor-image"
+              src="assets/images/sponsors/platinum/CJT Financial.png"
+            />
+          </div>
+        </div>
+      </div>
+      <div class="section">
+        <h2 class="section-title">Gold Tier</h2>
+        <div class="sponsors">
+          <div class="sponsor">
+            <h3 class="sponsor-name">The Woodlands SS</h3>
+            <img
+              class="sponsor-image"
+              src="assets/images/sponsors/gold/The Woodlands SS.png"
+            />
+          </div>
+          <div class="sponsor">
+            <h3 class="sponsor-name">Metro</h3>
+            <img
+              class="sponsor-image"
+              src="assets/images/sponsors/gold/Metro.png"
+            />
+          </div>
+        </div>
+      </div>
+      <div class="section">
+        <h2 class="section-title">Silver Tier</h2>
+        <div class="sponsors">
+          <div class="sponsor">
+            <h3 class="sponsor-name">Mississauga Cooksville Lions Club</h3>
+            <img
+              class="sponsor-image"
+              src="assets/images/sponsors/silver/Mississauga Cooksville Lions Club.png"
+            />
+          </div>
+          <div class="sponsor">
+            <h3 class="sponsor-name">East Port MFG</h3>
+            <img
+              class="sponsor-image"
+              src="assets/images/sponsors/silver/East Port MFG.png"
+            />
+          </div>
+          <div class="sponsor">
+            <h3 class="sponsor-name">Homelife Golconda Realty</h3>
+            <img
+              class="sponsor-image"
+              src="assets/images/sponsors/silver/Homelife Golconda Realty.png"
+            />
+          </div>
+          <div class="sponsor">
+            <h3 class="sponsor-name">JiuKun</h3>
+            <img
+              class="sponsor-image"
+              src="assets/images/sponsors/silver/JiuKun.png"
+            />
+          </div>
+        </div>
+      </div>
     </main>
 
-    <hr class="zbm">
-    <footer style="text-align: center;">
-        <a href="https://www.facebook.com/frc4308" style="color: #3b5998"><i class="bx bxl-facebook-square bx-lg"></i></a>
-        <a href="https://twitter.com/frcteam4308" style="color: #1da1f2"><i class="bx bxl-twitter bx-lg"></i></a>
-        <a href="https://www.youtube.com/channel/UCFOy9P3ZDVzclLQF3w-mtsw" style="color: #ff0000"><i class="bx bxl-youtube bx-lg"></i></a>
-        <a href="https://www.instagram.com/frc4308/" style="color: #c13584"><i class="bx bxl-instagram bx-lg"></i></a>
-        <a href="mailto:the.woodlands.robotics@gmail.com"><i class="bx bx-mail-send bx-lg"></i></a>
+    <footer class="footer">
+      <div>Made with ♡ by Team 4308 & vidhan</div>
```

Comment:
Can this entire line be removed as it looks unprofessional.

Reply:
Fixed with 639d8ce56fb62db6c4aa404b63b6c8e5a9a17bba.